src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,119 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-style: normal;
+  background-color: $page-color;
+}
+
+.card-catalog {
```

Comment:
inconsistency between the file name and BEM-block name
+ it's separate block - move it to own file